### PR TITLE
PX e2e changes o support new container service, portworx charts and multiregion 

### PR DESCRIPTION
**Pull request by mumutyal**

### CODE REVIEW

File: block-storage-attacher/tests/e2e/pvg/pvg-tester-common-functions.sh [hunk 1]
```diff
@@ -44,7 +44,7 @@ function add_worker_pools_and_zones {
     local zones_array=( `bx cs zones |tail -n +3` )
     for zone in "${zones_array[@]}"
     do
-        bx cs worker-pool-create --name pool_$zone --cluster $1 --machine-type $PVG_BX_MACHINE_TYPE --size-per-zone  $PVG_BX_CLUSTER_WORKERS_COUNT
+        bx cs worker-pool-create --name pool_$zone --cluster $1 --flavor $PVG_BX_MACHINE_TYPE --size-per-zone  $PVG_BX_CLUSTER_WORKERS_COUNT
```
Comment (akgunjal) [inline block-storage-attacher/tests/e2e/pvg/pvg-tester-common-functions.sh:47]: use `ibmcloud` here as well similar to other places

File: block-storage-attacher/tests/e2e/pvg/pvg-tester-common-functions.sh [hunk 2]
```diff
@@ -44,7 +44,7 @@ function add_worker_pools_and_zones {
...
         public_zone=$(get_public_vlan $zone)
         private_zone=$(get_private_vlan $zone)
         bx cs zone-add --zone $zone --cluster $1 --worker-pools pool_$zone --public-vlan $public_zone --private-vlan $private_zone
```
Comment (akgunjal) [inline block-storage-attacher/tests/e2e/pvg/pvg-tester-common-functions.sh:50]: use ibmcloud here as well similar to other places

File: block-storage-attacher/tests/e2e/scripts/common.sh [hunk 3]
```diff
@@ -38,14 +39,35 @@ function setKubeConfig {
...
-    cat $configfile
-    export KUBECONFIG=$configfile
+    cluster_id=$(ibmcloud ks clusters | awk "/$cluster_name/"'{print $2}')
+    echo "Generating Kube Config through 'ibmcloud ks  cluster config $cluster_name --admin' and exporting KUBECONFIG"
+    ibmcloud ks  cluster config --cluster $cluster_name --admin
+    kube_config_location="$bluemix_home/.bluemix/plugins/container-service/clusters/$cluster_name-$cluster_id-admin/kube-config*.yml" 
+    echo "$kube_config_location"
+    ls -l $kube_config_location
+  
+     if ls $kube_config_location 1> /dev/null 2>&1; then
```
Comment (akgunjal) [inline block-storage-attacher/tests/e2e/scripts/common.sh:59]: we dont need to do `export KUBECONFIG` step now with latest ibmcloud plugin. So remove this if-else checks. Just cluster config will download and set to current cluster.

File: block-storage-attacher/tests/e2e/scripts/common.sh [hunk 4]
```diff
@@ -295,10 +317,10 @@ function rm_cluster {
     removed=1
     cluster_name=$1
 
-    bx cs clusters
+    ibmcloud ks clusters
 
     for i in {1..3}; do
-        if bx cs cluster-rm $cluster_name -f; then
+        if ibmcloud ks cluster-rm $cluster_name -f; then
```
Comment (akgunjal) [inline block-storage-attacher/tests/e2e/scripts/common.sh:323]: This command is `ibmcloud ks cluster rm`

File: block-storage-attacher/tests/e2e/pvg/pvg-tester-common-functions.sh (hunk 1 above)
Comment (mumutyal) [reply block-storage-attacher/tests/e2e/pvg/pvg-tester-common-functions.sh:47]: We will not use PVG tester  functions

Comment (mumutyal) [reply block-storage-attacher/tests/e2e/pvg/pvg-tester-common-functions.sh:47]: For the VPC the cluster create procedure is entirely different  we will create with subnet and vpc id

File: block-storage-attacher/tests/e2e/pvg/pvg-tester-common-functions.sh (hunk 2 above)
Comment (mumutyal) [reply block-storage-attacher/tests/e2e/pvg/pvg-tester-common-functions.sh:50]: We will not use pvg tester functions for the portworx  e2e

File: block-storage-attacher/tests/e2e/scripts/common.sh (hunk 3 above)
Comment (mumutyal) [reply block-storage-attacher/tests/e2e/scripts/common.sh:59]: The exisitng framework works with KUBECONFIG envoroment varible only that is why setting this value and passing

File: block-storage-attacher/tests/e2e/scripts/common.sh (hunk 4 above)
Comment (mumutyal) [reply block-storage-attacher/tests/e2e/scripts/common.sh:323]: This also we are not suing changed this one

**Pull request merged**